javascript-functions/main.js

```diff
@@ -0,0 +1,37 @@
+function convertMinutesToSeconds(x) {
+  var secondsCalculated = x * 60;
+  console.log('Amount of seconds:', secondsCalculated);
+  return secondsCalculated;
+}
+
+convertMinutesToSeconds(10);
+
+function greet(name) {
+  console.log('Greeting:', 'Hey, ' + name);
+  return name;
```

Comment:
Whoops, this isn't returning the correct value. I would remove the `console.log` from your function code blocks, and just `console.log` the `return` where you are calling the function.